### PR TITLE
Create Music Album

### PR DESCRIPTION
**Pull request by saifullah767**
# In this Pr I have Implemented the Following Requirement

- Create MusicAlbum class in a separate .rb file.
- Create Genre class with an association to the Item class (in a separate .rb file).
- All MusicAlbum class properties visible in the diagram should be defined and set up in the constructor method.
- All Genre class properties visible in the diagram should be defined and set up in the constructor method.
- Implement methods:
    - add_item method in the Genre class
        - should take an instance of the Item class as an input
        - should add the input item to the collection of items
        - should add self as a property of the item object (by using the correct setter from the item object)
    - can_be_archived?() in the MusicAlbum class
        - should override the method from the parent class
        - should return true if parent's method returns true AND if on_spotify equals true
        - otherwise, it should return false

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -18,11 +20,19 @@ def initialize
              end
     @albums = []
     @games = []
+
+    @books = []
+    @music_albums = []
```
Comment (Santiago220991) [inline app.rb]: I think It is a good idea to erase lines 23, 24, and 25 cause are overwriting the values loaded from the JSON files.

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
# Changes Required :recycle: 

Hi, @saifullah767 good job so far.



No linter errors.✔️
Good code design and implementation. ✔️
Add all the requirements. ✔️

Please check the comments below.

- There is an issue with the feature to load the labels and books. It stopped working.  Please check the comment in the code.

File: app.rb (hunk 1 above)
Comment (saifullah767) [reply app.rb]: Thank you for highlighting this

**Review by Santiago220991 — APPROVED** [review]
# APPROVED 🎉 🎉 🎉 



Your project is complete! There is nothing else to say other than... it's time to merge it: :shipit: 

Congratulations!

No linter errors.✔️
Good code design and implementation. ✔️
Add all the requirements. ✔️

**Pull request merged**